### ISSUE TITLE
Error with jQuery noConflict mode

### ISSUE THREAD

**Issue by jasonhinkle** [julianshapiro/velocity#11]
The Page-wide options defaults are being set using `$` which breaks in jQuery noConflict mode.  I replaced:

```
$.fn.velocity.defaults = {
```

with

```
jQuery.fn.velocity.defaults = {
```

That seems to work find in noConflict mode as far as I can tell.

**Comment by janwerkhoven** [comment]
Hi I have to use `jQuery.noConflict` which causes `jQuery(...).velocity is not a function` errors.

```
var $hd = jQuery.noConflict();
...
$hd(...).velocity({...}) // results in "Velocity is not a function"
jQuery(...).velocity({...}) // results in "Velocity is not a function"
```

I'm loading velocity 1.2.3 before jQuery 1.11.2

Any ideas?

**Comment by kokarn** [comment]
You are loading velocity before jQuery, do it in the reverse order and you
should be fine.

**Comment by janwerkhoven** [comment]
Sorry that was a typo. I do load velocity after jQuery...

**Comment by kokarn** [comment]
We are using that at work with a jsDelivr link (https://cdn.jsdelivr.net/g/jquery@2.2.0,velocity@1.2.3) and then something like this

```javascript
ourGlobal.$ = $.noConflict();
jQuery = $;

( function( $ ){
    'use strict';

    $( '.js-element' ).velocity( 'fadeIn' );
})( ourGlobal.$ );
```

Can you show more of your code?

**Comment by janwerkhoven** [comment]
That's a good trick, that way I don't have to use `$hd` everywhere.

I replaced all `$hd` with plain jQuery `$` and I still keep getting "Velocity is not a function..." errors. The issue is thus not related to the `noConflict`. Probably a compilation issue?

**Comment by janwerkhoven** [comment]
Solved: Upgrade jQuery to v3.0.0 :+1:
